src/Command/SniffCommand.php

```diff
@@ -91,4 +91,21 @@ private function getFormatter(string $type, bool $pretty_format): FormatterInter
 
         return new ConsoleFormatter($pretty_format);
     }
+
+    private function guessStandard(?string $standard): string
+    {
+        if (null !== $standard) {
+            return $standard;
+        }
+
+        if (file_exists('csssniff.xml.dist')) {
+            return realpath('csssniff.xml.dist');
+        }
+
+        if (file_exists('csssniff.xml')) {
```

Comment:
Shouldn't the `csssniff.xml` file have preference over the `csssniff.xml.dist` file?

Reply:
I agree, this is the default way for phpunit and phpcodesniffer.

Reply:
ah I had them confused then. Will swap them.